.github/workflows/update-prod.yml

```diff
@@ -0,0 +1,27 @@
+name: Update Polkadot & Kusama Prod
+on:
+  schedule:
+    - cron: '0 0 * * *' # At the end of every day (00:00)
+  workflow_dispatch: # Or manually from GitHub UI
+
+jobs:
+  update_prod:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "18"
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set credentials
+        run: |
+          git config --global user.email "polkadot-kusama-automation-bot@users.noreply.github.com"
+          git config --global user.name "Polkadot-Kusama Bot"
+      - name: Rebase and deploy
+        run: |
+          git checkout --track origin/prod
+          git rebase master
+          git push
```

Comment:
Does the bot have permission to push to the repo (like w3fdeploy) ?

Reply:
Good question, by default the `GITHUB_TOKEN` should have write permissions but I don't think I have visibility into if the W3F org see if different permissions were applied -> https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

We can run it and see if it works or if we know w3fdeploy is better suited I can try swapping it.  It looks like w3fdeploy uses an access secret from the repo which may be granting it elevated permissions in some way.